src/_locales/pt_BR/messages.json

```diff
@@ -0,0 +1,755 @@
+    "message": "Capture somente a parte superior da tela"
+  },
+  "html_options_suspend_screen_capturing_entire_page": {
+    "message": "Capturar a página inteira"
+  },
+  "html_options_suspend_screen_capturing_tooltip_line1a": {
+    "message": "Em uma guia suspensa, a extensão poderá exibir uma visualização de imagem da"
+  },
+  "html_options_suspend_screen_capturing_tooltip_line1b": {
+    "message": "aparência da guia bem antes de ser suspensa."
+  },
+  "html_options_suspend_screen_capturing_tooltip_line2a": {
+    "message": "O recurso de captura de tela é experimental e pode causar um uso significativo da CPU, bem como"
+  },
+  "html_options_suspend_screen_capturing_tooltip_line2b": {
+    "message": "um aumento pequeno no uso de memória."
+  },
+  "html_options_suspend_screen_capturing_tooltip_line3a": {
+    "message": "Se você notar algum comportamento estranho, como as guias demorarem muito tempo para serem suspensas, ou se o Chrome"
+  },
+  "html_options_suspend_screen_capturing_tooltip_line3b": {
+    "message": "travar inesperadamente, tente desativar esse recurso."
+  },
+  "html_options_suspend_force_screen_capture": {
+    "message": "Ativar o modo de captura de tela de alta qualidade"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line1": {
+    "message": "Quando desativada, as capturas de tela têm as seguintes limitações para auxiliar no desempenho:"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line2": {
+    "message": "- Captura de tela de baixa qualidade"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line4": {
+    "message": "- Não irá capturar a tela se levar mais de 1 minuto para gerar a imagem"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line5": {
+    "message": "- A altura máxima de captura de tela é limitada a 5000px"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line6": {
+    "message": "Ativar o modo de captura de tela de alta qualidade pode aumentar significativamente a carga da CPU"
+  },
+  "html_options_suspend_force_screen_capture_tooltip_line7": {
+    "message": "e o uso de memória."
+  },
+  "html_options_whitelist_title": {
+    "message": "Nunca suspender as guias com URLs da lista a seguir:"
+  },
+  "html_options_whitelist_tooltip_line1": {
+    "message": "Adicione a URL de cada página que você deseja colocar na lista branca em uma nova linha. Por exemplo:"
+  },
+  "html_options_whitelist_tooltip_line2": {
+    "message": "Para colocar na lista branca vários sites em uma linha, você pode especificar parte da URL:"
+  },
+  "html_options_whitelist_tooltip_line3": {
+    "message": "Você também pode especificar expressões regulares colocando o texto em barras:"
+  },
+  "html_options_whitelist_test": {
+    "message": "Lista branca de teste"
+  },
+  "html_options_other_enable_context_menu": {
+    "message": "Adicionar o Grande Suspensor no menu de contexto do clique do botão direito do mouse"
```

Comment:
"message": "Adicionar The Great Suspender no menu de contexto do clique do botão direito do mouse"